src/tour.gleam

```diff
@@ -585,3 +615,26 @@ fn lesson_html(page: Lesson) -> String {
   |> htmb.render_page("html")
   |> string_builder.to_string
 }
+
+const theme_picker_js = "
+window.setDarkTheme = function() {
+  document.documentElement.classList.add('theme-dark')
+  document.documentElement.classList.remove('theme-light')
+  localStorage.setItem('theme', 'dark')
+};
+
+window.setLightTheme = function() {
+  document.documentElement.classList.add('theme-light')
+  document.documentElement.classList.remove('theme-dark')
+  localStorage.setItem('theme', 'light')
+};
+
+(function initTheme() {
+  const theme = localStorage.getItem('theme') || 'light'
+  if (theme == 'dark') {
+    window.setDarkTheme()
+  } else {
+    window.setLightTheme()
+  }
+})();
```

Comment:
Could you remove the iife and use a JS module instead please 🙏

Reply:
done